staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2507,6 +2537,12 @@ type ContainerStatus struct {
 	// Is always true when no startupProbe is defined.
 	// +optional
 	Started *bool `json:"started,omitempty" protobuf:"varint,9,opt,name=started"`
+	// Node compute resources allocated for the container.
+	// +optional
+	ResourcesAllocated ResourceList `json:"resourcesAllocated,omitempty" protobuf:"bytes,10,rep,name=resourcesAllocated,casttype=ResourceList,castkey=ResourceName"`
+	// Compute resource requests and limits enacted on the running container.
+	// +optional
+	Resources ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,11,opt,name=resources"`
```

Comment:
should be a pointer (optional)

Reply:
unresolved

Reply:
I'll take another stab at it tonight. This change taking a bit of work (I was nervous about the missed nil checks) and I wanted to get the rest of the changes out for review.

Reply:
Fixed in commit https://github.com/kubernetes/kubernetes/pull/102884/commits/1154796fb7923fc21db67e2483a12e31dda61810